build.gradle

```diff
@@ -3,20 +3,17 @@ plugins {
 }
 
 group 'io.cify'
-version '1.3.4'
+version '1.4.1'
```

Comment:
Is it an attempt to align versions with framework?

Reply:
No, it's a regular version increase. Any similarities to framework versions are purely coincidental=)